Src/library/net/abstract/socket/socket.e

```diff
@@ -1041,42 +1126,6 @@ feature {NONE} -- Implementation
 		deferred
 		end
 
-feature -- Socket Recv and Send timeout.
```

Comment:
to complete my reply ... all the timeout, blocking behavior and so on, are in NETWORK_SOCKET.
Since recv and send timeout makes sense when we have blocking behavior, I added the related feature in same class.

Now I can move the c externals to SOCKET.